src/main/java/ca/mcgill/mcb/pcingola/snpEffect/commandLine/SnpEffCmdEff.java

```diff
@@ -629,17 +644,29 @@ else if (outFor.equals("GATK")) {
 						} else if (outFor.equals("TXT")) usage("Output format 'TXT' has been deprecated. Please use 'VCF' instead.\nYou can extract VCF fields to a TXT file using 'SnpSift extractFields' (http://snpeff.sourceforge.net/SnpSift.html#Extract).");
 						else usage("Unknown output file format '" + outFor + "'");
 					}
-				} else if ((arg.equals("-s") || arg.equalsIgnoreCase("-stats"))) {
+				} else if (arg.equalsIgnoreCase("-htmlStats")) {
+					createSummaryHTML = true;
 					if ((i + 1) < args.length) {
-						summaryFile = args[++i];
-						String base = Gpr.baseName(Gpr.baseName(summaryFile, ".html"), ".csv"); // Extension can be either HTML or CSV
-						String dir = Gpr.dirName(summaryFile);
-						summaryGenesFile = (dir != null ? dir + "/" : "") + base + ".genes.txt";
+						summaryFileHTML = args[++i];
+                        if (summaryGenesFile.equals(DEFAULT_SUMMARY_GENES_FILE)) {
```

Comment:
This if condition only work when the default file name is used (doesn't make sense to me). Should be removed.